### ISSUE TITLE
Questions about Incorrectly labeled data in Object Detection Challenge

### ISSUE THREAD

**Issue by lydoc** [VIPriors/vipriors-challenges-toolkit#20]
Hello, I am interested in 2021 VIPriors Object Detection Challenge. However, I found that there are some problems with the annotations in the training data. For example, some bounding boxes of class 'steer' are incorrectly labeled in the position of the class 'front_pedal'. 

Can we choose to abandon this part of the wrong labeling information? Thanks.
@rjbruin @oskyhn

**Comment by oskyhn** [comment]
Hello, thank you for your message. We used crowd-sourcing for annotations, thus there might be a few wrong annotations in the training set. You can discard whichever images or parts you want.

**Comment by lydoc** [comment]
Thanks

**Comment by lydoc** [comment]
@oskyhn Hello, another question need to be confirmed. We visualized the bounding boxes in the training dataset, and found that the range of boxes in the same category is very different. For example, some bounding boxes of 'front_handbreak' are like this(48833.jpg) 
![image](https://user-images.githubusercontent.com/40634350/129138966-086eba4e-e6e7-4273-802e-1c6f34b2dcb3.png)
and some are like this (48915.jpg)
![image](https://user-images.githubusercontent.com/40634350/129139569-9069806c-b38d-422e-adc2-da81a524ba65.png)
(48957.jpg) 
![image](https://user-images.githubusercontent.com/40634350/129139630-1eec1886-e489-4bc4-95bf-b0c8ae129873.png)

We have to check with you, is this the same for the test set?? I mean the bounding boxes' ground truth.

**Comment by Zhang-HM** [comment]
Same question.

**Comment by yunh-w** [comment]
@oskyhn   Is the label of the test set consistent with the training set ？ Can I discard images or parts manually？
![image](https://user-images.githubusercontent.com/82502517/129195019-1bf252b3-7ede-4076-a50c-dda3aba74024.png)
![image](https://user-images.githubusercontent.com/82502517/129195052-61c8894e-8d2d-4ec2-ae73-dcfb00f1ff64.png)

**Comment by oskyhn** [comment]
> @oskyhn Hello, another question need to be confirmed. We visualized the bounding boxes in the training dataset, and found that the range of boxes in the same category is very different. For example, some bounding boxes of 'front_handbreak' are like this(48833.jpg)
> ![image](https://user-images.githubusercontent.com/40634350/129138966-086eba4e-e6e7-4273-802e-1c6f34b2dcb3.png)
> and some are like this (48915.jpg)
> ![image](https://user-images.githubusercontent.com/40634350/129139569-9069806c-b38d-422e-adc2-da81a524ba65.png)
> (48957.jpg)
> ![image](https://user-images.githubusercontent.com/40634350/129139630-1eec1886-e489-4bc4-95bf-b0c8ae129873.png)
> 
> We have to check with you, is this the same for the test set?? I mean the bounding boxes' ground truth.

Hello,
As far as I can see in the cropped images, the **front_handbreaks** are correctly located in images **48915.jpg** and **48957.jpg**: If I follow the cables, they indicate front side.
In the image **48833.jpg**, **front_handbreak** might have the object state as **absent**. We also find out some parts which are labelled like this with an object state of absent. Absent parts are already discarded for training and testing. 

@yunh-w: If you want to discard some parts or images, please feel free to do that. We would like to see more data efficient solution in the workshop.

The testset has been checked and most of the issues were fixed, but it may still contain a few overlooked labels.

**Comment by lydoc** [comment]
@oskyhn Actually, in the train_annotations.json, the front_handbreak in 48833.jpg has the object state as occluded. I visualized some results in training set and I think there are lots of inaccurate labeling. lol....
48680.jpg 
![image](https://user-images.githubusercontent.com/40634350/129325955-89500d35-3f8d-4dcb-a5d8-b0afecb0827e.png)
48684.jpg
![image](https://user-images.githubusercontent.com/40634350/129326018-e598c6f8-36fe-4a30-8c75-821da41e916a.png)
48687.jpg
![image](https://user-images.githubusercontent.com/40634350/129326068-12ac9de4-556d-4c2d-901e-2b0e7dcacd94.png)
48911.jpg
![image](https://user-images.githubusercontent.com/40634350/129326139-493df6a5-2070-47d5-9bba-f7021ee3cf35.png)
49642.jpg
Almost half of the data has this problem...
